### ISSUE TITLE
hook to android's release process

### ISSUE THREAD

**Issue by lukin0110** [lukin0110/poeditor-gradle#1]
When you run `gradle assembleRelease` for Android the plugin should download the translations. Cfr crashlytics: https://github.com/crashlytics/gradle-plugin-example

**Comment by f6v** [comment]
Are you accepting PR on this?

**Comment by lukin0110** [comment]
Of course. Feel free to fork & open a pull request. Once your pull request is ready notify me via mail maarten[AT]lukin.be.
